Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -964,6 +964,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Basic-Car-Maintenance/Preview Content\"";
+				DEVELOPMENT_TEAM = 34P32RF9MT;
```

Comment:
Please remove this! Look at the [contributing guidelines](https://github.com/mikaelacaron/Basic-Car-Maintenance/blob/dev/CONTRIBUTING.md#start-here) again, because you've added your development team here, when it should be in the xcconfig file instead!